deployments/dev/environment.py

```diff
@@ -43,7 +43,7 @@ def env() -> Mapping[str, Optional[str]]:
         ]),
 
         'AZUL_TDR_SOURCES': ','.join([
-            'tdr:broad-jade-dev-data:snapshot/hca_dev_20201203___20210524_lattice:'
```

Comment:
We're adding that snapshot to the catalog, it doesn't replace the existing snapshot. The issue description wasn't clear on this so I won't bump.